public/preload.js

```diff
@@ -0,0 +1,12 @@
+const { ipcRenderer } = require('electron');
+const backend = require('i18next-electron-fs-backend');
+
+// NOTE: if enable contextIsolation, you need to change the code like below
```

Comment:
Currently, we can not enable contextIsolation because of errors below.
But we need to enable it in someday, so I left a NOTE to pass i18nextElectronBackend api in contextIsolation enabled environment.

![image](https://user-images.githubusercontent.com/941296/95479332-9a7ca780-09c5-11eb-88df-d5809eb66b92.png)

Reply:
I think the preload is not necessary, this can also be done in the renderer thread, right?

Reply:
Ah... I think this can  be done in the renderer thread technically, but we should not do that.

We need to do both set `nodeIntegration` `false` and set `contextIsolation` `true` in the future for security reason.
And currently both are un-secure setting (`nodeIntegration` is set true explicitly in `electron.js`, `contextIsolation` is default disabled in older version than electron 12)

If we set `nodeIntegration` false, render thread would not be able to use nodejs features. So this approach, load file in preload.js and use it from render thread, would be necessary to build secure electron app in the near future.

related link
https://www.electronjs.org/docs/tutorial/security#isolation-for-untrusted-content